GANDLF/cli/deploy.py

```diff
@@ -256,14 +264,19 @@ def get_metrics_mlcube_config(mlcube_config_file, entrypoint_script):
     return mlcube_config
 
 
-def get_model_mlcube_config(mlcube_config_file, requires_gpu, entrypoint_script):
+def get_model_mlcube_config(
+    mlcube_config_file: str, requires_gpu: bool, entrypoint_script: str = None
```

Comment:
entrypint_script should be annotated Union[str,None]

Reply:
Fixed with `Optional`.